### PR TITLE
Calculator: Disallow divide by zero

### PR DESCRIPTION
**Pull request by pawlos**
Divide by zero was allowed in the current Calculatort implementation. This simple fix displays an error in case of a division by zero and resets the calculator state.

### CODE REVIEW

**Comment by byteduck** [comment]
LGTM, thank you! Once I get the chance to test it I'll merge.

File: programs/applications/calculator/CalculatorWidget.cpp
Comment (byteduck) [inline programs/applications/calculator/CalculatorWidget.cpp:156]: Looks like this error message is a little long to fit in the default size of the calculator window. Could we change this to something like `NaN`?

**Review by byteduck — CHANGES_REQUESTED** [review]

File: programs/applications/calculator/CalculatorWidget.cpp
Comment (pawlos) [reply programs/applications/calculator/CalculatorWidget.cpp]: @byteduck updated to NaN.

**Comment by byteduck** [comment]
Thank you, merged! 🙂

**Pull request merged**
